### PR TITLE
LongBow

### PR DESCRIPTION
**Pull request by DenaUxUi**
LongBow from Dena_

## About the PR

Added LongBow without craft recipe
Добавлен Длинный Лук без рецепта крафта

## Why / Balance

Added new weapon
Добавлено новое оружие

## Media

https://github.com/user-attachments/assets/d14247c1-7cd9-4452-9110-f49e1a9d195e

### CODE REVIEW

File: Resources/Prototypes/_CP14/Entities/Objects/Weapons/Ranged/longBow.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+- type: entity
+  id: CP14LongBow
+  name: LongBow
+  parent: BaseItem
+  description: Long-bow
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Objects/Weapons/Ranged/longBow.yml:5]: Change description to more interesting please

File: Resources/Prototypes/_CP14/Entities/Objects/Weapons/Ranged/longBow.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+- type: entity
+  id: CP14LongBow
+  name: LongBow
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Objects/Weapons/Ranged/longBow.yml:3]: no uppercase in names

**Review by TheShuEd — CHANGES_REQUESTED** [review]

File: Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/inhand-left.png
Comment (Jaraten) [inline Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/inhand-left.png:1]: sprite in hand doesn't look good, better to tweak it for perspective.
Снимок экрана 2024-08-01 113344

File: Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/inhand-right.png
Comment (Jaraten) [inline Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/inhand-right.png:1]: Same fault as above

File: Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/unwielded-arrow.png
Comment (Jaraten) [inline Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/unwielded-arrow.png:1]: arrow sprite looks flat, add shadows and lights

File: Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/wielded-inhand-left.png
Comment (Jaraten) [inline Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/wielded-inhand-left.png:1]: image
the form is very strange and messy

File: Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/wielded-inhand-right.png
Comment (Jaraten) [inline Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/wielded-inhand-right.png:1]: > the form is very strange and messy

same

File: Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/equipped-BACKPACK.png
Comment (Jaraten) [inline Resources/Textures/_CP14/Objects/Weapons/Ranged/longbow.rsi/equipped-BACKPACK.png:1]: image
back sprite looks incomplete.

**Review by Jaraten — CHANGES_REQUESTED** [review]
Work on light shading and shape.

**Comment by TheShuEd** [comment]
@DenaUxUi, вы планируете продолжать работу над этим?

**Pull request closed without merging**